flake.nix

```diff
@@ -26,19 +26,30 @@
             '';
           };
 
-          bobadatabase = pkgs.writeShellApplication {
-            name = "bobadatabase";
-            runtimeInputs = [ pkgs.postgresql_12 ];
+          bobadatabase = {
```

Comment:
This isn't valid, packages have to be a flat set, they can't contain nested objects :(

You can use `nix flake check` to run checks and also `nix flake show` to print out the outputs.

Reply:
Huh! That's funny cause it works correctly when I used it (but `nix flake check` does error).

Is there a way to keep these scripts together somehow since they're related?

Reply:
You can name them similarly, it's not an issue if they're two separate packages exposed on the flake.